tests/modules/test_closing.py

```diff
@@ -11,16 +12,41 @@ def test_run(image, module, image_set, workspace):
     module.y_name.value = "closing"
 
     if image.dimensions == 3 or image.multichannel:
+        # test 3d structuring element
         module.structuring_element.shape = "ball"
 
         selem = skimage.morphology.ball(1)
+
+        module.run(workspace)
+
+        actual = image_set.get_image("closing")
+
+        desired = skimage.morphology.closing(image.pixel_data, selem)
+
+        numpy.testing.assert_array_equal(actual.pixel_data, desired)
+
+        # test planewise
+        selem = skimage.morphology.disk(1)
+
+        module.run(workspace)
```

Comment:
Before `module.run(workspace)` you'll need to specify the new structuring element:

```python
module.structuring_element.shape = "disk"

module.run(workspace)
```